src/portfolios/portfolio/Portfolio.vue

```diff
@@ -19,6 +19,23 @@
                   <span class="text-base-dark">Last Sync: Nov. 15, 0100</span> 
                 -->
               </div>
+              <div class="d-flex justify-space-between width-100 mb-6">
+                <ATATAlert
+                    id="InaccurateFinancialDetails"
+                    type="error"
+                    class="container-max-width my-10"
+                >
+                  <template v-slot:content>
+                    <h3 class="mb-6">Financial Details may be inaccurate</h3>
+                    <p class="mb-0">
+                      We are currently experiencing an issue retrieving cost data from {{cspName}}
+                      In the meantime, administrators can login to your CSP console directly to
+                      get detailed cost analyses and breakdowns.
+                      We apologize for this inconvenience.
+                    </p>
+                  </template>
+                </ATATAlert>
+              </div>
```

Comment:
This alert has many issues. Please refer to figma.
https://www.figma.com/file/6zwE1QbRrJZ3yFuA0bo7he/ATAT-Portfolio?type=design&node-id=335-44048&mode=dev

In general, always copy/paste text from Figma instead of typing. This would have prevented the cap D and the missing word "with".

CSP name is using abbreviation. Please use the "long" names I slacked you.
```  
public serviceProvider: Record<string, string> = {
    AZURE: "Microsoft Azure",
    AWS: "Amazon Web Services",
    GCP: "Google Cloud",
    ORACLE: "Oracle Cloud"
  }
  ```

![Screen Shot 2023-07-05 at 9 29 37 AM 1](https://github.com/dod-ccpo/atat-web-ui/assets/90333349/03af8044-9b05-4456-bc64-223911fc20d1)